packages/zone.js/lib/common/events.ts

```diff
@@ -394,6 +404,11 @@ export function patchEventTarget(
         const passive =
             passiveSupported && !!passiveEvents && passiveEvents.indexOf(eventName) !== -1;
         const options = buildEventListenerOptions(arguments[2], passive);
+        const signal = typeof options === 'object' && options?.signal;
```

Comment:
Consider updating the below to 
```suggestion
       const signal = options && typeof options === 'object' && options.signal && typeof options.signal === 'object' ? options.signal : undefined;
```

To avoid the repetitive checks... 
```ts
if (typeof signal === 'object' && signal?.aborted) 
if (signal && typeof signal === 'object')
```

Reply:
Got it, updated, thanks.